### ISSUE TITLE
Npmjs

### ISSUE THREAD

**Issue by caceres97** [nullcc/ts-transformer-keys-2#1]
Can you upload your amazing job to npmjs.com please? :)

**Comment by vamidi** [comment]
Yes please. trying to add this to my project, because I need this really bad! :D

**Comment by Luabee** [comment]
I too would like to sign the NPM petition.

**Comment by nullcc** [comment]
Hi, @caceres97 @vamidi @Luabee , please use https://github.com/nullcc/ts-interface-keys-transformer which had been uploaded to the NPM registry. 
Project "ts-transformer-keys-2" will be archived
